mathics/core/list.py

```diff
@@ -86,6 +86,10 @@ def __str__(self) -> str:
 
     # @timeit
     def evaluate_elements(self, evaluation):
```

Comment:
Suggested signature: 

```
    def evaluate_elements(self, evaluation: Evaluation) -> Expression:
```

Yes, I realize this was from before. I am trying to up our game in terms of quality incrementally as we improve things.